2021-07-03 08:16:17,110 - gravity_core.wlogger - ERROR - Traceback (most recent call last):
  File "/home/qodex/Gravity_core_export/gravity_core/wlistener.py", line 102, in broadcastMsgSend
    conn.send(msg)
ConnectionResetError: [Errno 104] Connection reset by peer

2021-07-03 08:16:17,113 - gravity_core.wlogger - ERROR - Traceback (most recent call last):
  File "/home/qodex/Gravity_core_export/gravity_core/wlistener.py", line 102, in broadcastMsgSend
    conn.send(msg)
ConnectionResetError: [Errno 104] Connection reset by peer

2021-07-03 08:16:17,161 - gravity_core.wlogger - ERROR - Traceback (most recent call last):
  File "/home/qodex/Gravity_core_export/gravity_core/wlistener.py", line 102, in broadcastMsgSend
    conn.send(msg)
BrokenPipeError: [Errno 32] Broken pipe

2021-07-03 08:16:17,164 - gravity_core.wlogger - ERROR - Traceback (most recent call last):
  File "/home/qodex/Gravity_core_export/gravity_core/wlistener.py", line 102, in broadcastMsgSend
    conn.send(msg)
BrokenPipeError: [Errno 32] Broken pipe

2021-07-03 08:16:17,179 - gravity_core.wlogger - ERROR - Traceback (most recent call last):
  File "/home/qodex/Gravity_core_export/gravity_core/wlistener.py", line 102, in broadcastMsgSend
    conn.send(msg)
BrokenPipeError: [Errno 32] Broken pipe

2021-07-03 08:16:17,180 - gravity_core.wlogger - ERROR - Traceback (most recent call last):
  File "/home/qodex/Gravity_core_export/gravity_core/wlistener.py", line 102, in broadcastMsgSend
    conn.send(msg)
BrokenPipeError: [Errno 32] Broken pipe

